core/integration/features/auctions/5460-market-order-cannot-trade-due-to-auction.feature

```diff
@@ -69,7 +69,7 @@ Feature: Test for issue 5460
 
     Then the market data for the market "ETH/DEC21" should be:
       | mark price | trading mode                    | horizon | min bound | max bound | target stake | supplied stake | open interest |
-      | 110000000  | TRADING_MODE_MONITORING_AUCTION | 43200   | 82757690  | 122741898 | 84115906     | 100000000      | 1410607       |
+      | 110000000  | TRADING_MODE_MONITORING_AUCTION | 43200   | 82757664  | 122741858 | 84107975     | 100000000      | 1410474       |
```

Comment:
they're sort of independent, but the reference price ends up being different because the LP order sizes are slightly different because of the probability of trading so the weighted average ends up slightly different.